src/utils/configdata/configs.ts

```diff
@@ -1,3 +1,72 @@
-const configs = {};
+const configs = {
+  searchCategories: {
```

Comment:
It should look like below:
{
  pages : {
    home_page : ["header_title","search_input","search_categories"]
  }
  elements: {
    header_title : { type: "text", style: { color: brand.primary, height: dim.standard } }
    search_input: {
      
    }
  }
}

- The mentioned dim will be another slice, may be dimSlice which contains all the dimensions mentioned in urils/configdata/dim.ts file
- Like this file, everything in the component/screen should be dynamic and controlled from global store. no hardcoded values should be used anywhere.